presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -198,6 +199,23 @@ protected void assertInvalidCast(String projection, String message)
         }
     }
 
+    protected void assertNotSupported(String projection, Type expectedType, String message)
```

Comment:
Nevermind, I see why this is needed. Though it's a bit strange, since it's different from the others in a non-obvious way.